### Geofrenzy GfDwellBoolset message
#
# A dwell message specifies the current entitlement.
#
# The boolean set dwell ... 
#
Header      header      # Standard ROS header
GfEntHeader ent_header  # Entitlement header
int32       bitset_num  # Boolean bit set

